src/Plugins/PluginInfo.php

```diff
@@ -3,10 +3,14 @@
 namespace Pantheon\Terminus\Plugins;
 
 use Consolidation\AnnotatedCommand\CommandFileDiscovery;
+use League\Container\ContainerAwareInterface;
+use League\Container\ContainerAwareTrait;
 use Pantheon\Terminus\Exceptions\TerminusException;
 
-class PluginInfo
+class PluginInfo implements ContainerAwareInterface
```

Comment:
You're right. That's a relic of a change I started on and then abandoned. Thanks for catching it.